File: Codeforces/Golang/800-1000/1669A - Division/main.go
package main

import (
	"fmt"
)

func main() {
	var t, rating int
	fmt.Scan(&t)
	for i := 0; i < t; i++ {
		fmt.Scan(&rating)
		if 1900 <= rating {
			fmt.Println("Division 1")
		} else if 1600 <= rating && rating <= 1899 {
			fmt.Println("Division 2")
		} else if 1400 <= rating && rating <= 1599 {
			fmt.Println("Division 3")
		} else if rating <= 1399 {
			fmt.Println("Division 4")
		}
	}
}

File: Codeforces/Rust/800-1000/1669A - Division/src/main.rs
use std::io;

fn main() {
    let mut input = String::new();
    let _ = io::stdin().read_line(&mut input);

    let t: i64 = input.trim().parse().unwrap();

    for _ in 0..t {
        input.clear();
        let _ = io::stdin().read_line(&mut input);

        let rating: i64 = input.trim().parse().unwrap();

        if 1900 <= rating {
            println!("Division 1")
        } else if 1600 <= rating && rating <= 1899 {
            println!("Division 2")
        } else if 1400 <= rating && rating <= 1599 {
            println!("Division 3")
        } else if rating <= 1399 {
            println!("Division 4")
        }
    }
}
